addons/sale/models/product_category.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields
+
+ACCOUNT_DOMAIN = "['&', ('deprecated', '=', False), ('account_type', 'not in', ('asset_receivable','liability_payable','asset_cash','liability_credit_card','off_balance'))]"
```

Comment:
dubious... should be able to thin that down: that domain won't remove half of the accounts

Reply:
Used the same domain than the one used by 
![image](https://github.com/user-attachments/assets/d20e38fa-788e-41c1-990a-9e38acebc564)
Which are the two fields in the product category view 
![image](https://github.com/user-attachments/assets/0993be86-41f5-4793-8d82-2aee259f910e)

I can restrict the domain a bit more if you want ?

Reply:
oh, I suppose you convinced me :)